themeable-variables.json

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
Do we want to check this in with the dev-site?

Reply:
probably not, but it annoyingly get generated.

Reply:
yeah...